Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -59,10 +60,13 @@ const Double_t kmaxChi2PerCluster = 20.;
 const Double_t kmaxChi2PerTrack = 30.;
 // Tracking "road" from layer to layer
 const Double_t kRoadY = 0.2;
-const Double_t kRoadZ = 0.7;
+const Float_t kRoadZ = 0.3;
```

Comment:
do you want to change also `kRoadY` to `float` for consistency? Maybe you can also add a comment about the changed value. Is this because the inclination of the track is taken into account to calculate an effective value?

Reply:
In fact, all these parameters should now in O2 become "float".  (This is because our "base track" is made of floats).  Fixed as well !
Yes, the initial value was changed, because this "road" now becomes a function of the
track inclination angle.